### DISCUSSION TITLE
how to allow a path not to be authenticated

### DISCUSSION THREAD

**Discussion by vijaypothireddy** [OpenIDC/mod_auth_openidc#697] [General]
I am trying to bypass authentication for  path starting wit /src and for all other i wanted to use auth. below is sample config but i when i request https://sample-user12.sample.test.com/src/containers/base/view.js i still get 401 instead of file . can some one explain what i am doing wrong ?

   <IfModule mod_ssl.c>
   <VirtualHost *:443>
    ..........

        SSLProxyEngine On
        ProxyPass "/" "https://xx.xx.xx.xx/"
        ProxyPassReverse "/" "https://xx.xxx.x.x/"
        ProxyPreserveHost On
        SSLProxyVerify none 
        SSLProxyCheckPeerCN off
        SSLProxyCheckPeerName off
        SSLProxyCheckPeerExpire off

        OIDCProviderMetadataURL https://auth.test.com/.well-known/openid-configuration
        OIDCClientID lJCW5w0VA8azgq1VY0vvQyoXZMFpDKPs
        OIDCClientSecret 'fMgINnWzYcqnEWRXvpsXxdO495WLsgWqc3EdiozXxZ5IBZci7Ia1wCkADIlf-ejP'
        OIDCDefaultURL https://test.com
	OIDCDefaultLoggedOutURL https://test.com
        OIDCScope "openid name email"
        OIDCRedirectURI https://sample-user12.sample.test.com/redirect_uri/
        OIDCSessionInactivityTimeout 1200
        OIDCSessionMaxDuration  18000
        OIDCStateMaxNumberOfCookies 10 true
        OIDCCryptoPassphrase 'ttf9TsZDuhRxs6dzjee7t0WVtoKmQPj8Oy5VpulRGFfc4cUzPERcFpysdtIWtCP'

        <LocationMatch /src>
           OIDCUnAuthAction pass
        </LocationMatch>        

        <Location />
          AuthType openid-connect
          Require claim https://sample/company:jbkjhihk
          OIDCUnAuthAction 401 "%{REQUEST_URI}  !~ m#^\/([^\/]+\.html)?\/?$#"
        </Location>
	

</VirtualHost>
</IfModule>

**Comment by zandbelt** [comment]
According to the Apache docs `Location` directives are processed in the order that they appear in the config file and the last one wins. so you'll need to switch those.
